global/overlay/etc/cosmos/apt/bootstrap-cosmos.sh

```diff
@@ -1,6 +1,7 @@
 #!/bin/sh
 
-set -e
+#set -e 
+# not all breakage is un-recoverable...
```

Comment:
Better if those places that want to recover from errors ought to do `foo || echo "foo failed, continuing"` or set +e ... set -e IMO.

Reply:
Resolved in latest commit, thanks for the review!